src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.figure;
+
+public abstract class Figure {
+    private String color;
+
+    protected Figure(String color) {
+        this.color = color;
+    }
+
+    protected String getColor() {
+        return color;
+    }
+
+    public abstract void print();
```

Comment:
let's move this method to the separate interface Drawable and rename print() to draw(), Figure should implement this interface

Reply:
what about this comment?